reporting/impl/src/main/resources/reports/templates/migration-issues.ftl

```diff
@@ -223,37 +222,7 @@
             }
         });
     });
-
-   	    function resizeTables()
-        {
```

Comment:
After testing it definitely does... but I assumed this was intentional and I guess it was just a mistake. Oops. :)